marketplace-and-integrations/README.md

```diff
@@ -8,15 +8,28 @@ description: >-
 
 Within this Umbraco DXP (Digital Experience Platform) area of the Umbraco documentation, you'll find topics related to extending Umbraco CMS via packages and integrations.
 
-Umbraco is used in solutions in both a "platform" and "composable" DXP context. With the former, Umbraco has responsibility for both content management and delivery, with extensions added as NuGet packages. In a composable DXP, Umbraco is often part of an overall architecture, focussed on the management of content exposed via APIs for delivery. In both situations, backoffice integrations are an important part of providing an intuitive, unified editor experience.
+Umbraco is used in solutions in both a "platform" and "composable" DXP context. With the former, Umbraco has responsibility for both content management and delivery, with extensions added as NuGet packages. In a composable DXP, Umbraco is often a stand-alone service that takes part in an overall architecture. In these cases, Umbraco CMS is focused on the management of content exposed via APIs for delivery.
 
-Our aim is to enhance the "platform DXP", support the CMS in a "composable DXP", and make it simpler to build integrations with Umbraco.
+Umbraco is also used in a hybrid DXP context. Here, some values are added to the Umbraco platform, like Umbraco Forms or Umbraco Commerce, where Umbraco acts as a component in a wider architecture. In all situations, backoffice integrations are an important part of providing an intuitive, unified editor experience.
+
+Our aim is to support the different ways of using Umbraco in a DXP context and provid resources for building integrations with Umbraco.
+
+![Umbraco as a platform DXP, composable DXP og hybrid solution](.gitbook/assets/pDXPcDXP.png)
+
+
+### Packages and integrations  
+Both packages and integrations are defined as Umbraco Packages that can be either [Nuget packages (Umbraco 10+) or packages zip files (Umbraco 8 or earlier)](https://docs.umbraco.com/umbraco-cms/extending/packages/types-of-packages).
+Integrations provide functionality via a third-party service's API, whereas packages extend the functionality of the CMS (management or delivery).
```

Comment:
```suggestion
Packages for Umbraco 10+ are installed via [Nuget packages](https://docs.umbraco.com/umbraco-cms/extending/packages/types-of-packages).

Integrations provide functionality via a third-party service's API, whereas packages extend the functionality of the CMS (management or delivery).
```

I'm not sure we really need to be calling out Umbraco 8 features any more, so think we can remove this part.

Reply:
If we're not mentioning Umbraco 8, does it then make sense to mention Umbraco 10+?
We could just leave out the version numbers entirely.